recipes/drm4g/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "drm4g" %}
+{% set version = "3.0.11" %}
+{% set sha256 = "ece23c282d1d8fb82b34a8885166676984fd846da7ff644edc5fc174364de2a0" %}
+{% set build_number = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/drm4g-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k or win or osx]
```

Comment:
```suggestion
  skip: true  # [not linux]
```

Reply:
py2k was added by grayskull, the 'or win or osx'

Reply:
Yes, that’s fine. This is just a quicker way of specifying it but the result is the same.